index.html

```diff
@@ -742,7 +742,10 @@ <h3>Validate Algorithm</h3>
         </li>
         <li>
 Let |status| be the value at the position indicated by the
-|credentialIndex| times the |size| in the |bitstring|.
+|credentialIndex| times the |size| in the |bitstring|. If the |credentialIndex|
+is a value outside of the range of the |bitstring|, a
```

Comment:
This confuses me. `|credentialIndex| times the |size|` must be within `the range of the |bitstring|`, since that's what `indicated` `the position` of the `|status|` `value`. The `|credentialIndex|` `value` itself doesn't appear to have any relation to `the range of the |bitstring|`.

I think the above should change to either
```
Let |status| be the value in the |bitstring| at the position indicated
by the |credentialIndex| times the |size|. If the |credentialIndex|
times the |size| is a value outside of the range of the |bitstring|, a
```
or
```
Let |status| be the value in the |bitstring| at the position indicated
by the |credentialIndex|. If the |credentialIndex| is a value outside
of the range of the |bitstring|, a
```

This is not made as a GitHub suggestion, because a line "deleted" (really, changed) by the PR (line 745) is enveloped in the three I'm changing (lines 744-746).

Reply:
Fixed in 1d1a83a8eee4415203fe88c71a65b77b96986cdc.